config/guidance_document_collections.json

```diff
@@ -0,0 +1,687 @@
+[
+  {
+    "base_path": "/government/collections/national-curriculum-assessments-information-for-parents",
+    "surface_collection": false,
```

Comment:
(I assume `false` isn't technically required, but is here for clarity/emphasis?)